v2/api/containerservice/v1api20230201/storage/compat/istio_components_types_gen.go

```diff
@@ -1,105 +0,0 @@
-// Code generated by azure-service-operator-codegen. DO NOT EDIT.
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-package compat
-
-import (
-	v20231001s "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20231001/storage"
```

Comment:
Any idea why these files were deleted?

Reply:
Allocating types to files is based on root types - and we were only treating Resources + ARM Status + ARM Spec as available roots, so every type in the `compat` packge ended up in a separate file. My tweaks in `reference_graph.go` add "otherwised unreferenced types" as potential roots, consolidating things in to fewer files.